nix/pkgs/haskell/extra.nix

```diff
@@ -98,6 +102,7 @@ let
   updateAllShaFiles = writeShellScript "updateShaFiles" ''
     ${updateShaFile cabalInstallProject ./cabal-install.sha}
     ${updateShaFile agdaProject ./agda.sha}
+    ${updateShaFile agdaProject.projectCross.mingwW64 ./agda-windows.sha}
```

Comment:
Shouldn't this just happen by us running this with cross platforms in `updateMaterialized`?

Reply:
A checked and without this line running:
```
nix-shell --run updateMaterialized
```
does not update `agda-windows.sha`